migrations/versions/2023_07_09_21_52_50_7ae4658467d7_change_created_utc_to_datetimez_for_.py

```diff
@@ -0,0 +1,48 @@
+"""Change created_utc to datetimez for commentflags
+
+Revision ID: 7ae4658467d7
+Revises: ea282d7c711c
+Create Date: 2023-07-09 21:52:50.386177+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '7ae4658467d7'
+down_revision = 'ea282d7c711c'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('commentflags', sa.Column('created_datetimez', sa.DateTime(timezone=True), nullable=True, server_default=sa.text('NOW()')))
```

Comment:
the alembic script can be made simpler like so

for each table do
```py
    op.alter_column(table_name, 'created_utc', new_column_name='created_datetimez')
    op.execute(f"ALTER TABLE {table_name} ALTER COLUMN created_datetimez TYPE timestamp with time zone USING to_timestamp(created_datetimez);")
```

where `table_name` is the name of the table